prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -6,9 +6,21 @@ postsubmits:
     spec:
       containers:
       - image: gcr.io/gcp-guest/osconfig-agent-build:latest
-        imagePullPolicy: Always
         command:
-        - "/main.sh"
+        - "/daisy_build_upload.sh"
+        env:
+        - name: PROJECT
+          value: "gcp-guest"
+        - name: ZONE
+          value: "us-central1-c"
+        - name: BASE_REPO
+          value: "GoogleCloudPlatform"
+        - name: REPOSITORY
```

Comment:
prow will set some var for github details - plus they do the checkout already

Reply:
removed env variables and passing the values as args